configure

```diff
@@ -39,16 +39,13 @@ sub_error()
     exit 1
 }
 
+# NOTE: Always quote your warning() messages
 warning()
 {
-    echo
-    echo "Warning: $*"
-}
-
-sub_warning()
-{
-    echo "  ... $*"
-    echo
+    >&2 echo
+    >&2 echo "Warning: $1"
+    [ -n "${2:-}" ] && >&2 echo " ... $2" # Built-in sub_warning()
+    >&2 echo
 }
```

Comment:
Why combine `warning` with `sub_warning` but not `error` with `sub_error`? This is inconsistent.

Reply:
Mainly because you can use sub_error() with info() without error() but warning() (and sub_warning()) didn't used like this.

Reply:
This seems borderline to me, but ok.